community/values/src/main/java/org/neo4j/values/storable/PointValue.java

```diff
@@ -220,7 +220,8 @@ public String toString()
 
     public String toIndexableString()
     {
-        return format( "Point{%s, %s}", getCoordinateReferenceSystem().getName(), Arrays.toString( coordinate ) );
+        CoordinateReferenceSystem crs = getCoordinateReferenceSystem();
+        return format( "%d-%d%s", crs.getTable().getTableId(), crs.getCode(), Arrays.toString( coordinate ) );
```

Comment:
Since all spatial types are converted using the same key, we should differentiate on the sub-types here.